src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,172 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int PLACE_FOR_NULL_KEY = 0;
+    private static int capacity;
+    private static int size;
+    private static int threshold;
+    private static int index;
+    private Node<K, V>[] table;
+    private Node<K, V> currentNode;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        capacity = DEFAULT_INITIAL_CAPACITY;
+        threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+        size = 0;
```

Comment:
we dont need to initialize size by 0, as it will be initialized by 0 "by default"